blog/_posts/btcpay-server-1-5-0.md

```diff
@@ -0,0 +1,117 @@
+
+We'd love to hear what do you think about the dashboard and how it works for you. Let us know what kind of data would you help you better manage your store.
+
+## Payout processor 🤖
+
+A payout processor is a new concept hooked into the [pull payment system], specifically the [payout part of it]. This new functionality allows you to automate the flow of the payouts.
+
+![](/images/1.5.0-payout-processor-1.png)
+
+Currently, we support payout processors for on-chain and off-chain payment methods. You can schedule payouts for on-chain, bolt11, lightning address, lnurl, etc.
+
+Set up a time interval in your store settings upon which your payouts will be batched and broadcasted to the network. 
+
+![](/images/1.5.0-payout-processor-2.png)
+
+Payout processors in a current shape and form allows you to save on transaction fees by batching the store's payments (wallet payments, refunds, payouts) and broadcasting them at a specific, pre-set time interval, for example, hourly, daily, weekly and more.
+
+### Schedule for later 📆
+
+We understand that the payout processors may sound a bit abstract, and that's why we've added a concrete use-case for it through our brand-new **schedule for later** function in the wallet.
+
+![](/images/1-5-0-schedule-transaction1.png)
+
+This means that now, instead of signing and broadcasting a transaction, you have the option to schedule it for later on at a pre-set time interval.
+
+And since our refunding system goes through the wallet, the scheduling option now allows you to batch your customer's refunds and save on fees.
+
+![](/images/1-5-0-schedule-transaction2.png)
+
+We plan to introduce a better permission system and hook it into the store so that in a particular store role, the store manager can create payouts, for example, refunds, but only the store owner can send them out.
+
+## Enable/disable users
+
+Server admins now have the options to disable or re-enable users, allowing them to better manage people who use their instance. 
+
+![](/images/1-5-0-enable-disable-users.png)
+
+This functionality can be handy for third-party hosts or services building on top of BTCPay Server or offering BTCPay third-party hosting.
+
+## NBXplorer updates 🔎
+
+In case you've missed it, our dependency NBXPlorer has undergo a major performance refactoring. 
+
+Most of the refactoring was done to cather the way data is handled in the dashboard, but also to improve the performance for larger enterprise users.
+
+You can read more about the NBXplorer updates [in this blog post](https://blog.btcpayserver.org/nbxplorer-postgres/).
+
+## UX/UI improvements 🎨
+
+As usually, with every release we're ironing out the user-interface, making incremental changes based on our understanding of how you use BTCPay Server. Your feedback is always critical factor when we're making such changes.
+
+- Better **lightning services access** - access lightning network services in a click of a button.
```

Comment:
```suggestion
- Better **Lightning services access** - access Lightning Network services with the click of a button.
```

Reply:
Is there really standardization for this? In Bitcoin Design we lowercase all to avoid confusion.

Reply:
I mostly find it uppercased, as a specific term - e.g. the [Bitcoin wiki](https://en.bitcoin.it/wiki/Lightning_Network) and [Bitcoin Optech](https://bitcoinops.org/en/newsletters/2022/04/20/) refer to it as "Lightning Network".